pkg/dhtnode-common.pkg

```diff
@@ -15,8 +15,11 @@ OPTS.update(
 
 # Replace existing args without rebinding
 ARGS[:] = FUN.mapfiles('deploy/logrotate', '/etc/logrotate.d',
```

Comment:
I'm actually a bit surprised about this syntax, I think in the docs it's mentioned to use `ARGS.extend()`. I guess I probably did it myself, and is not introduced here, but it might be worth updating at some point.